src/executor.cpp

```diff
@@ -30,6 +30,8 @@ void Executor::displayExecutionStatus(
     }
 
     convert << (result ? " 00" : " FF");
+
+    storeDbusExecStatus(result, convert.str(), "");
```

Comment:
Why this function is required? You have a class member variable for this just call std::make_tuple with the values and assign that in place.